src/screens/login/login-page.tsx

```diff
@@ -1,22 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {  View,  Text, TouchableOpacity, TextInput } from 'react-native';
 import { styles } from './login-page.styles';
+import { notEmpty, emailValid, passwordLength, passwordHasNumber, passwordHasLetter} from './login-validation';
 
-export function LoginPage(){
+export const LoginPage = () => {
+
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [errorMessage, setErrorMessage] = useState('');
+    const [hasError, setHasError] = useState(false);
+
+    const handleSubmit = ()  => {
+        if (notEmpty(email, password) === false){
```

Comment:
Tem uma simplificação que podemos fazer nas igualdades booleanas no Javascript:

```suggestion
        if (!notEmpty(email, password)) {
```

Tecnicamente são comparações um pouco diferentes, porém na prática o resultado seria o mesmo.